webapp/README.MD

```diff
@@ -0,0 +1,34 @@
+# Running webapp Instructions:
+
+## For MAC OS/Linux:
+- Install pip3
+    ```sh
+    sudo apt-get install python3-pip
+    ```
+
+- Install virtualenv
+    ```sh
+    pip3 install virtualenv
+    ```
+
+- Change Directory
+    ```sh
+    cd <project-path>/devops-students/webapp
+    ```
+
+- Create Virtual Env
+    ```sh
+    virtualenv -p python3 venv
+    ```
+
+- Activate Virtual Env
+    ```sh
+    source venv/bin/activate
+    ```
+
+- Install Project Requirements
```

Comment:
When you finish setting up your virtualenv and installing the requirements on it, read the Python Flask quickstart to give you a bit of an introduction to Flask. The Quickstart should help us get started with rendering our HTML templates
https://flask.palletsprojects.com/en/2.2.x/quickstart/